atd-etl/cris_import/cris_import.py

```diff
@@ -612,7 +612,23 @@ def align_records(map_state):
             input_column_names = util.get_input_column_names(pg, map_state["import_schema"], table, target_columns)
 
             # iterate over each imported record and determine correct action
+            should_skip_update = False
+
             for source in imported_records:
+                # Check unique key columns to make sure they all have a value
+                for key_column in key_columns:
+                    key_column_value = source[key_column]
+                    if key_column_value == None:
+                        print("\nSkipping because unique key column is missing")    
```

Comment:
Super open to whatever amount of logging we want here. 🙏 I used this because it works good for testing.

Reply:
this looks good to me. maybe add another `print` here that prints the entire `source`, so that we have some hope of tracking this record down?

Reply:
Mike pointed out that we don't want to print people records to our logs. i retract this suggestion 🙏

Reply:
chatted with John about this and decided to keep as is so we don't print out sensitive people data. 🔒